src/features/pages/deriv-ez/hero/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { hero_images } from './data'
+import FlexBox from 'features/components/atoms/flex-box'
+import Typography from 'features/components/atoms/typography'
+import TradeHeroContainer from 'features/components/templates/hero-banners/trade'
+import useBreakpoints from 'components/hooks/use-breakpoints'
+import { Localize } from 'components/localization'
+
+const DerivEZHero = () => {
+    const { is_mobile, is_mobile_or_tablet } = useBreakpoints()
```

Comment:
[visibility.scss](https://github.com/binary-com/deriv-com/blob/translation-beta/src/features/styles/theme/visibility.scss) would be good instead of this.